WMF Framework/CommonStrings.swift

```diff
@@ -54,6 +54,10 @@ public class CommonStrings: NSObject {
     
     @objc public static let notificationsCenterTitle = WMFLocalizedString("notifications-center-title", value: "Notifications", comment: "Title for Notifications Center interface, as well as the accessibility label for the button that navigates to Notifications Center.")
     @objc public static let notificationsCenterBadgeTitle = WMFLocalizedString("notifications-center-badge-button-accessibility-label", value: "Notifications with unread badge", comment: "Accessibility label for a button that navigates to Notifications Center. This button has a badge indicating there are unread notifications.")
+    
+    @objc public static let profileButtonTitle = WMFLocalizedString("profile-button-accessibility-label", value: "Profile", comment: "Accessibility label the profile navigation bar button. Tapping it navigates to the user profile view.")
```

Comment:
Since they're navigation buttons, should we have something in the value itself that says like "Navigate to profile"?

Reply:
profileViewButtonItem.accessibilityLabel = WMFLocalizedString("profile-button", value: "Double tap to view profile", comment: "Accessibility label for opening the profile sheet")

Reply:
@l-olson1214 I added an accessibility hint to these buttons, it took me a bit of time to track down a related bug. Let me know what you think about this approach. Per Apple docs:

> An accessibility hint helps users understand what happens when they perform an action on the accessibility element when that result isn't obvious from the accessibility label.